### PR TITLE
artisan arabic

### LINKED ISSUE #6: Translate file "artisan.md"
Notes for contributors, the file can be translated by being aware of and understanding the file you are translating, you must have an understanding of the techniques and the explanation provided by the documentation file, and it must be in Standard Arabic, and the technical terms should be translated as follows:
For example, المتحكمات with the term next to it in English (Controllers)
For example, التهجير and next to it the term in English (Migrations)
And so on
Thank you for your contribution

### PR DESCRIPTION
**Pull request by lumi-hussien**

### CODE REVIEW

File: artisan.md [hunk 1]
```diff
@@ -1,122 +1,155 @@
...
-    - [Calling Commands From Other Commands](#calling-commands-from-other-commands)
-- [Signal Handling](#signal-handling)
-- [Stub Customization](#stub-customization)
-- [Events](#events)
-
-<a name="introduction"></a>
-## Introduction
-
-Artisan is the command line interface included with Laravel. Artisan exists at the root of your application as the `artisan` script and provides a number of helpful commands that can assist you while you build your application. To view a list of all available Artisan commands, you may use the `list` command:
+#وحدة تحكم الحرفي الماهر
```
Comment (karam-mustafa) [inline artisan.md:1]: please keep the English phrase

**Comment by karam-mustafa** [comment]
@lumi-hussien  Hello lumi
have you finished this file ?

**Comment by lumi-hussien** [comment]
Hi karam
Yes, I have finished

File: artisan.md [hunk 2]
```diff
@@ -1,122 +1,155 @@
-# Artisan Console
-
-- [Introduction](#introduction)
-    - [Tinker (REPL)](#tinker)
```
Comment (karam-mustafa) [inline artisan.md:4]: Kepp the valid links, in this line and all the links below

File: artisan.md [hunk 3]
```diff
@@ -1,122 +1,155 @@
...
+REPL (read-eval-print-loop) نوع من الواجهات السوداء التفاعلية التي تأخذ مدخلات مستخدم وحيد و يقيم المدخلات و يعيد النتيجة للمستخدم
+مشغلة بواسطة حزمة (https://github.com/bobthecow/psysh)  PsySH
+ 
 
-<a name="installation"></a>
-#### Installation
+<a name="التنصيب"></a> 
+#### التنصيب
+
+التنكر مضمن افتراضيا بتطبيقات اللارافل لكن يمكنك استخدام Composer لتنصيبه في حال حذفه من التطبيق
```
Comment (karam-mustafa) [inline artisan.md]: keep "tinker" instead of "تنكر"

File: artisan.md [hunk 4]
```diff
@@ -1,122 +1,155 @@
...
+للبحث عن واجهة رسومية للتفاعل مع تطبيق لارافل 
+راجع الرابط https://tinkerwell.app
+
 
-<a name="usage"></a>
-#### Usage
+<a name="الاستخدام"></a> 
+#### الاستخدام
+
+التنكر يسمح لك التفاعل مع تطبيق اللارافل في واجهة الأوامر و المودلات و الاعمال و الاحداث
```
Comment (karam-mustafa) [inline artisan.md]: Replace all works "tinker" with "تنكر", because it is the name of the package, not the term for the technical

**Review by karam-mustafa — CHANGES_REQUESTED** [review]

**Comment by lumi-hussien** [comment]
ok

**Pull request closed without merging**